src/peergos/shared/user/App.java

```diff
@@ -118,6 +119,18 @@ private Path fullPath(Path path) {
         });
     }
 
+    @JsMethod
+    public CompletableFuture<List<String>> dirInternal(Path relativePath) {
+        Path path = fullPath(relativePath);
+        return ctx.getByPath(path).thenCompose(dirOpt -> {
```

Comment:
there is a getChildren call on usercontext

Reply:
I tried that, but the version on user context blows up if the path does not exist (at least it does from gwt js). The above returns an empty list.

Reply:
ok let's fix it to cover that case